src/AzureRenderManager/WebApp/Arm/AzureResourceProvider.cs

```diff
@@ -116,7 +153,7 @@ private IEnumerable<string> GetClassicAdministratorEmails(List<ClassicAdministra
 
             var accessToken = await GetAccessToken();
             var token = new TokenCredentials(accessToken);
-            var authClient = new AuthorizationManagementClient(token) { SubscriptionId = subscriptionId.ToString() };
+            var authClient = new AuthorizationManagementClient(token, _httpClientFactory.CreateClient(), false) { SubscriptionId = subscriptionId.ToString() };
```

Comment:
note that you don't need to specify "don't Dispose" everywhere as the clients returned by `IHttpClientFactory` have their real lifetimes managed by the factory, Disposing them doesn't do much (except cancel any existing requests)

Reply:
also: should we be disposing all these `AuthorizationManagementClient`s, or is the only resource they have the `HttpClient`?

Reply:
Unfortunately it's a required parameter when specifying the HttpClient.  I believe that's the only resource being help by that client, although we should probably go thorugh at some point and make sure they're all being disposed.